feature/src/main/java/com/gdg/feature/calendar/CalendarRoute.kt

```diff
@@ -1,51 +1,394 @@
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                Row(verticalAlignment = Alignment.CenterVertically) {
+                    Text(
+                        text = stringResource(R.string.calender_people_reporting_title),
+                        style = CrowdZeroTheme.typography.c3Regular,
+                        color = CrowdZeroTheme.colors.gray600
+                    )
+                    Spacer(modifier = Modifier.width(4.dp))
+                    Text(
+                        text = data.people,
+                        style = CrowdZeroTheme.typography.c3Regular,
+                        color = CrowdZeroTheme.colors.gray800
+                    )
+                }
+
+                Spacer(modifier = Modifier.width(8.dp))
+
+                Row(verticalAlignment = Alignment.CenterVertically) {
+                    Text(
+                        text = stringResource(R.string.calender_jurisdiction),
+                        style = CrowdZeroTheme.typography.c3Regular,
+                        color = CrowdZeroTheme.colors.gray600
+                    )
+                    Spacer(modifier = Modifier.width(4.dp))
+                    Text(
+                        text = data.jurisdiction,
+                        style = CrowdZeroTheme.typography.c3Regular,
+                        color = CrowdZeroTheme.colors.gray800
+                    )
+                }
+            }
+        }
     }
 }
 
+@Composable
+fun CalendarComponent(
+    currentMonth: YearMonth,
+    selectedDate: LocalDate,
+    onMonthChange: (YearMonth) -> Unit,
+    onDateSelected: (LocalDate) -> Unit,
+) {
+    val days = remember(currentMonth) { generateDaysForMonth(currentMonth) }
+
+    Column(
+        modifier = Modifier
+            .fillMaxWidth()
+            .padding(20.dp),
+        horizontalAlignment = Alignment.CenterHorizontally
+    ) {
+        // 상단: 년/월 & < > 버튼
+        Row(
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(horizontal = 5.dp),
+            verticalAlignment = Alignment.CenterVertically,
+            horizontalArrangement = Arrangement.Center
+        ) {
+            Text(
+                text = stringResource(
+                    R.string.calender_component_date,
+                    currentMonth.year.toString(),
+                    currentMonth.monthValue.toString()
+                ),
+                style = CrowdZeroTheme.typography.h3Bold,
+                color = CrowdZeroTheme.colors.green700
+            )
+
+            Spacer(modifier = Modifier.width(225.dp))
+
+            Icon(
+                painter = painterResource(id = R.drawable.calender_button_l),
+                tint = CrowdZeroTheme.colors.green700,
+                contentDescription = "Next Month",
+                modifier = Modifier
+                    .size(16.dp)
+                    .clickable { onMonthChange(currentMonth.minusMonths(1)) }
```

Comment:
P1: noRippleClickable로 수정해야 리플 없어집니다
```suggestion
                    .noRippleClickable { onMonthChange(currentMonth.minusMonths(1)) }
```